pkg/libxdploader/libxdp_loader.go

```diff
@@ -1,3 +1,6 @@
+//go:build cgo
+// +build cgo
```

Comment:
do we need both or just `// +build cgo` is sufficient ?

Reply:
it's gofmt that adds it automatically